test/common.js

```diff
@@ -10,7 +10,7 @@ const password = '72297d971271bc62ca899bba7432acb1';
 const apiKey = 'bc731e500840231da5b43bb3f388d2f0';
 const shopName ='johns-apparel';
 
-const shopify = new Shopify(shopName, accessToken);
+const shopify = new Shopify({shopName, accessToken});
```

Comment:
Sorry to pester, keep a space before and after (`{ shopName, accessToken }`) and do this also for all other cases.
I should definitely add more linting rules :)